internal/config/config.go

```diff
@@ -83,7 +83,7 @@ func New(provider string) (h config.Handler, err error) {
 
 	// Setup Operations Config
 	if err := h.SetObject(adapter.OperationsKey, Operations); err != nil {
-		return nil, err
+		return nil, adapter.ErrClientConfig(err)
```

Comment:
@leecalcote Is it ok now?

Reply:
@MUzairS15 thanks for this. Yes, closer. There should be definition of long description, probable cause, and suggested remediation, too.

Reply:
> @MUzairS15 thanks for this. Yes, closer. There should be definition of long description, probable cause, and suggested remediation, too.

BTW, @leecalcote what abt this?